app/src/main/java/com/kickstarter/ui/activities/HelpActivity.java

```diff
@@ -30,8 +30,9 @@
   public static final int HELP_TYPE_HOW_IT_WORKS = 2;
   public static final int HELP_TYPE_COOKIE_POLICY = 3;
   public static final int HELP_TYPE_FAQ = 4;
```

Comment:
HELP_TYPE_HELP_CENTER is replacing HELP_TYPE_FAQ, so we wouldn't need both here.